src/Cake.Common/Tools/NuGet/List/NuGetListSettings.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.NuGet.List
+{
+    /// <summary>
+    /// Contains settings used by <see cref="NuGetList"/>.
+    /// </summary>
+    public sealed class NuGetListSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether list all versions of a package.By default, only the latest package version is displayed
+        /// </summary>
+        public bool AllVersions { get; set; }
+
+        /// <summary>
+        /// Gets or sets the amount of details in the output: normal, quiet, detailed.
+        /// </summary>
+        public NuGetVerbosity? Verbosity { get; set; }
```

Comment:
Correct me if I'm wrong, but wont setting Verbosity  to either Quiet or Detailed mess up parsing? The parser expects one package per line with `id` and `version` separated by whitespace. Quiet verbosity won't output anything, while detailed will place them on separate rows plus additionally output some other noise.

Reply:
yip, for consistency one should explicitly be set.

Reply:
Explicitly setting the Verbosity as that indeed broke the parsing.